website/docs/d/kms_ciphertext.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "alicloud"
```

Comment:
please add a subcategory in the line 2: subcategory: "KMS", like other docs.

Reply:
done